src/test/java/uk/gov/ons/ssdc/supporttool/testhelper/IntegrationTestHelper.java

```diff
@@ -40,28 +54,37 @@
   public IntegrationTestHelper(
       SurveyRepository surveyRepository,
       CollectionExerciseRepository collectionExerciseRepository,
+      PrintTemplateRepository printTemplateRepository,
+      SmsTemplateRepository smsTemplateRepository,
+      CaseRepository caseRepository,
+      UacQidLinkRepository uacQidLinkRepository,
       UserRepository userRepository,
       UserGroupRepository userGroupRepository,
       UserGroupMemberRepository userGroupMemberRepository,
       UserGroupPermissionRepository userGroupPermissionRepository) {
     this.surveyRepository = surveyRepository;
     this.collectionExerciseRepository = collectionExerciseRepository;
+    this.printTemplateRepository = printTemplateRepository;
+    this.smsTemplateRepository = smsTemplateRepository;
+    this.caseRepository = caseRepository;
+    this.uacQidLinkRepository = uacQidLinkRepository;
     this.userRepository = userRepository;
     this.userGroupRepository = userGroupRepository;
     this.userGroupMemberRepository = userGroupMemberRepository;
     this.userGroupPermissionRepository = userGroupPermissionRepository;
   }
 
   public void testGet(
-      int port, UserGroupAuthorisedActivityType activity, BundleFunction bundleFunction) {
-    BundleOfUsefulTestStuff bundle = getTestBundle();
+      int port, UserGroupAuthorisedActivityType activity, BundleUrlGetter bundleUrlGetter) {
     setUpTestUserPermission(activity);
+    BundleOfUsefulTestStuff bundle = getTestBundle();
 
-    String url = String.format("http://localhost:%d/api/%s", port, bundleFunction.getUrl(bundle));
+    String url = String.format("http://localhost:%d/api/%s", port, bundleUrlGetter.getUrl(bundle));
     ResponseEntity<String> response = restTemplate.getForEntity(url, String.class);
     assertThat(response.getStatusCodeValue()).isBetween(200, 299);
```

Comment:
Ok older code but like the testPost this code this could be something like:   assertThat(response.getStatusCodeValue()).as("Testing GET").isEqualTo(200);
No need for a wide match here, and info is good.